src/main/java/uk/nhs/tis/sync/dto/SiteDmsDto.java

```diff
@@ -1,26 +0,0 @@
-package uk.nhs.tis.sync.dto;
-
-import com.fasterxml.jackson.annotation.JsonInclude;
-import com.fasterxml.jackson.annotation.JsonInclude.Include;
-import lombok.Data;
-
-@Data
-@JsonInclude(Include.NON_NULL)
-public class SiteDmsDto {
```

Comment:
As per mapper, the source has a nested DTO. There are only 2 records _without_ an endDate in the past.